packages/cli/src/test-utils/setup.global.ts

```diff
@@ -1,6 +1,6 @@
 import baseSetup from '@celo/dev-utils/lib/ganache-setup'
 // Has to import the matchers somewhere so that typescript knows the matchers have been made available
-import '@celo/dev-utils/lib/matchers'
+import _unused from '@celo/dev-utils/lib/matchers'
```

Comment:
note to reviewer: this caused the celocli tests to fail because `@celo/dev-utils` would not find jest. Weird.